src/helpers/randomizeCard/index.js

```diff
@@ -0,0 +1,736 @@
+import { FACTIONS, RACES, TYPES, RARITIES } from '~/constants/game'
+import arrayRandom from '~/helpers/arrayRandom'
+
+//##############
+//# To-Do List #
+//##############
+
+// Add more effects
+// Add effects that target structures
+// Make Satyr self-base destruction less prevalent
+// Add Dragons, Pirates, Felines, and plain Elders/Ancients
+// Add better filters - maybe even filter for effect classes?
+// Add effect classes for different types of effects
+// Add randomly generated names - maybe combo off of effect classes
+// Fix inconsistencies:
+// -- Affecting bordering/surrounding units with no bordering/surrounding units
+// -- Spawning units behind when played bordering the base
+// -- Tweak Convert targeting/conditions a bit - add a new category for Convert targets, and increase the number of conditions
+// -- Fix unspend mana calculations
+
+//################################
+//# Settings to mess around with #
+//################################
+
+// Turns on verbose logging, displaying how each card is calculated.
+var verbose = false 
+
+// Removes the caps on minimum mana, maximum mana, and effect cost. Can make some funy units.
+var uncapped = false 
```

Comment:
So we’d typically use `const` for variables that don’t get reassigned, or `let` otherwise instead of `var`. I could also enforce that with a linting rule if that’s more convenient.